docs/advanced-features/react-18/server-components.md

```diff
@@ -0,0 +1,125 @@
+# React Server Components (Alpha)
+
+Server Components allow us to render React components on the server. This is fundamentally different from server-side rendering (SSR) where you're pre-generating HTML on the server. With Server Components, there's **zero client-side JavaScript needed,** making page rendering faster. This improves the user experience of your application, pairing the best parts of server-rendering with client-side interactivity.
```

Comment:
```suggestion
Server Components allow us to render React components on the server. This is fundamentally different from server-side rendering (SSR) where you're pre-generating HTML on the server. With Server Components, there's **zero client-side JavaScript needed**, making page rendering faster. This improves the user experience of your application, pairing the best parts of server-rendering with client-side interactivity.
```

Also, should we be more clear about "zero client-side JavaScript"? Because there are necessary JS that will be shipped such as React's runtime, so technically it's not "zero" and it confuses people ([ref](https://github.com/vercel/next-rsc-demo/issues/11#issuecomment-988128075)).

I'll try to rephrase it as

```diff
- With Server Components, there's zero client-side JavaScript needed
+ Server Components won't increase your bundle size and ship zero JavaScript to the client
```